src/Console/Commands/DefaultCommand.php

```diff
@@ -64,16 +65,26 @@ protected function execute(InputInterface $input, OutputInterface $output): int
     protected function configure(): void
     {
         $this->setDescription('Checks for misspellings in the given directory.');
+
+        $this->addOption(
+            'dir',
+            'd',
+            InputOption::VALUE_OPTIONAL,
+            'The directory to check for misspellings.'
+        );
     }
 
     /**
      * Infer the project's base directory from the environment.
      */
-    private function inferProjectPath(): string
+    private function inferProjectPath(InputInterface $input): string
```

Comment:
Personally I wouldn't check this in this method, as it changes the intended functionality.

Probably it would be better to handle the option at the start of the execute() method and only call the inferProjectPath as a default if no value was passed.

Reply:
Yes - I see your point just by looking at the above diff, and reading the method's comment, I'll look at this later today - thanks